Feat  memstore return instructed status code (#419)
Layr-Labs/eigenda-proxy: store/generated_key/memstore/README.md

  curl -X PATCH http://localhost:3100/memstore/config -d @-
```

### Instructed Status Code Return

> The instructed status code return allows users to set a desired returned status code for some payloads the user is about to write. The next time when a user requests to get the payloads with keys, the proxy returns the result corresponding to the status code set earlier. The status code is sticky, and

Reviewer: wdym by "user is about to write"? Isn't there no write? You just set this option and then reads return this error.

can affect all subsequent writes. By default, the memstore isinitialized without the instructed status code return.

```bash
 curl -X PATCH http://localhost:3100/memstore/config -d '{"GetReturnsInstructedStatusCode": {"GetReturnsStatusCode": 3, "IsActivated": true }}'
 {"MaxBlobSizeBytes":2048,"BlobExpiration":"45m0s","PutLatency":"0s","GetLatency":"0s","PutReturnsFailoverError":false,"GetReturnsInstructedStatusCode":{"GetReturnsStatusCode":3,"IsActivated":true}}
```

A user can only activate the instructed status code via http PATCH method above. A user can switch to other status code from an existing activated status code

> by sending a new `PATCH` request, the GET for subsequent writes contains the new status code.

Reviewer: ```suggestion
by sending a new `PATCH` request. Subsequent GET requests will return the new status code.
```
```suggestion
by sending a new `PATCH` request, the GET for subsequent writes contains the new status code.
```
